templates/iot/examples/infinispan-service.yml

```diff
@@ -0,0 +1,80 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    description: Hot Rod's port.
+  labels:
+    application: infinispan-server
+  name: infinispan-server-hotrod
+spec:
+  ports:
+  - port: 11222
+    targetPort: 11222
+  selector:
+deploymentConfig: infinispan-server
+---
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  labels:
+    application: infinispan-server
+  name: infinispan-server
+spec:
+  replicas: 1
+  serviceName: infinispan-server-headless
+  template:
+    metadata:
+      labels:
+        application: infinispan-server
+        deploymentConfig: infinispan-server
+      name: infinispan-server
+    spec:
+      containers:
+      - args:
+        - cloud
+        - -Djboss.default.jgroups.stack=kubernetes
+        env:
+        - name: KUBERNETES_LABELS
+          value: application=infinispan-server
+        - name: KUBERNETES_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        image: jboss/infinispan-server:9.4.6.Final
+        livenessProbe:
+          exec:
+            command:
+            - /usr/local/bin/is_running.sh
+          failureThreshold: 5
+          initialDelaySeconds: 10
+          periodSeconds: 60
+          successThreshold: 1
+          timeoutSeconds: 80
+        name: infinispan-server
+        ports:
+        - containerPort: 8080
+          name: http
+          protocol: TCP
+        - containerPort: 9990
+          name: management
+          protocol: TCP
+        - containerPort: 8888
+          name: ping
+          protocol: TCP
+        - containerPort: 11222
+          name: hotrod
+          protocol: TCP
+        readinessProbe:
+          exec:
+            command:
+            - /usr/local/bin/is_healthy.sh
+          failureThreshold: 5
+          initialDelaySeconds: 10
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 80
+        resources:
+          requests:
+            cpu: "0.5"
+            memory: 512Mi
+terminationGracePeriodSeconds: 120
```

Comment:
I think this last line should be indented.

Reply:
fixed :)